basemaps/localization.md

```diff
@@ -66,133 +116,97 @@ Extending our Switzerland example with exonym and endonym from other languages:
 - `name:en` = `Switzerland`
 - `name:es` = `Switzerland`
 - `name:pt` = `Suíça`
-- `name:zh` = `瑞士`
 - `name:zh-Hans` = `瑞士`
 - `name:zh-Hant` = `瑞士`
 - _... many other localized values..._
 
-_NOTE: The Chinese (`zh`) examples above demonstrates how a single language can have multiple writing systems, in this case both simplified Chinese (`zh-Hans`) used in mainland China and tranditional Chinese (`zh-Hant`) used in Taiwan. The value stored in `zh` could be either of those._
```

Comment:
Restore the `zh` note, please.

Reply:
If we always normalize `zh` in OSM to the two explicate variants (as is indicated in https://github.com/protomaps/docs/pull/51#discussion_r1737939967), then this can be dropped.

Reply:
At the moment we only export `name:zh-Hant` and `name:zh-Hans` from OpenStreetMap to the tiles. If one or both of these are missing on the OSM feature, but `name:zh` is available, then we backfill `name:zh` into `name:zh-Hans` or `name:zh-Hant`.

If I remember correctly, you had a technique to say if a `name:zh` string was written in `name:zh-Hans` or `name:zh-Hant` in tilezen. Is that correct? If yes, how did you do it?